app-frontend/src/app/pages/library/projects/detail/projectScenes/projectScenes.html

```diff
@@ -14,57 +14,54 @@ <h1 class="h3 page-title">
                 ng-show="!$ctrl.$parent.selectedScenes.size && !$ctrl.isEditingProjectName && !$ctrl.isSavingProjectNameEdit">
           <i class="icon-pencil"></i>
         </button>
-        <div class="color-danger" ng-show="$ctrl.showError">
+        <div class="color-danger" ng-if="$ctrl.showError">
           There was an error renaming the project.
           <button class="btn btn-tiny" type="submit"
                   ng-click="$ctrl.dismissErrorMessage()">
             <i class="icon-cross color-danger"></i>
           </button>
         </div>
-        <div ng-show="$ctrl.isEditingProjectName">
+        <div ng-if="$ctrl.isEditingProjectName">
           <form class="ng-pristine ng-valid">
             <div class="form-group all-in-one">
               <input id="name" type="text" class="form-control"
                     placeholder="Edit project name" ng-model="$ctrl.editedProjectName"
                     ng-attr-autofocus="{{$ctrl.isEditingProjectName || undefined }}">
               <button class="btn btn-link" type="submit"
                       ng-click="$ctrl.cancelProjectNameEdit()"
-                      ng-show="!$ctrl.isSavingProjectNameEdit">
+                      ng-if="!$ctrl.isSavingProjectNameEdit">
                 <i class="icon-cross"></i>
               </button>
               <button class="btn btn-link" type="submit"
                       ng-click="$ctrl.saveProjectNameEdit()"
-                      ng-show="!$ctrl.isSavingProjectNameEdit">
+                      ng-if="!$ctrl.isSavingProjectNameEdit">
                 <i class="icon-check"></i>
               </button>
             </div>
           </form>
         </div>
+        <div class="flex-fill"></div>
+        <a href ng-click="$ctrl.$state.go('browse', {projectid: $ctrl.projectId})"
```

Comment:
It's odd that this takes you to Browse but that clicking "Edit" in the dropdown on the Projects list takes you to Color Correction. Since there's already a link to Browse on this page (Add Scenes in the sidebar), I think this should go to Color Correction.